de/manual/ch_GUIMenus.xml

```diff
@@ -6658,535 +6658,1179 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
-                </para>
-              </entry>
-            </row>
+                    <para>&mc-gmi.fi.prin-vou;
+                    </para>
+                  </entry>
 
-            <row>
-              <entry>
-                <para>&gmi.bus.cus.pro-pay;
-                </para>
-              </entry>
+                  <entry namest="c2" nameend="c3">
+                    <para>In einem neuen Fenster wird die aktuelle Rechnung bzw. Auslagenerstattung zum Drucken angezeigt.
+                    </para>
+                  </entry>
+                </row>
+              </tbody>
+            </tgroup>
+          </table>
+        </sect4>
+
+        <sect4 id="gui-business-invoice-edit-menu">
+          <title>Geschäft - &gm.edit;-Menü</title>
+
+          <para>Die <emphasis>Menüleiste</emphasis> im Menü &gm.edit; umfasst neben den in
+            <xref linkend="gui-window-edit-menu"/> beschriebenen Einträge noch zusätzlich die
+            folgenden Menüpunkte.
+          </para>
 
-              <entry>
-                <para>Den Dialog für die Verarbeitung einer Zahlung öffnen.
-                </para>
-              </entry>
-            </row>
-          </tbody>
-        </tgroup>
-      </table>
+          <table pgwide="1" frame="topbot" id="gui-business-invoice-editmenu">
+            <title>Geschäft - Bearbeiten-Menü - Bearbeitungsfunktionen für Rechnungen sowie Einstellungen</title>
 
-      <note>
-        <para>Das Kontextmenü erscheint, wenn Sie mit der rechten Maustaste auf einen Kunden in der Liste
-          klicken.
-        </para>
-      </note>
-    </sect2>
+            <titleabbrev>Geschäft - Bearbeiten-Menü</titleabbrev>
 
-    <sect2 id="business-vendor-overview">
-      <title>Lieferantenübersicht</title>
+            <tgroup align="left" cols="3">
+              <colspec colname="c1"></colspec>
 
-      <para>Um die Anzeige der <guilabel>Lieferantenübersicht</guilabel> zu öffnen, klicken Sie auf
-        &mc.bus.ven.o-view; .
-      </para>
+              <colspec colname="c2"></colspec>
 
-      <table pgwide="1" id="business-vendor-overview-toolbar-table">
-        <title>Lieferanten - Werkzeugleiste</title>
+              <colspec colname="c3"></colspec>
 
-        <tgroup cols="2">
-          <thead>
-            <row>
-              <entry>
-                <para>Schaltfläche
-                </para>
-              </entry>
+              <thead>
+                <row>
+                  <entry>
+                    <para>Menüpunkt
+                    </para>
+                  </entry>
 
-              <entry align="center">
-                <para>Beschreibung
-                </para>
-              </entry>
-            </row>
-          </thead>
+                  <entry align="center" namest="c2" nameend="c3">
+                    <para>Beschreibung
+                    </para>
+                  </entry>
+                </row>
+              </thead>
```

Comment:
Frage: Macht es Sinn den Header in ein entity zu packen? ich glaube, den haben wir öfter.